management/nodes/Create-VM.ps1

```diff
@@ -30,6 +30,7 @@ param (
     [string]$ImageVHD = $null,
     [ValidateSet('Windows','Linux')]
     [string]$OperatingSystem = $null,
+    [ValidateLength(0, 14)]
```

Comment:
This is the case only for WIndows VMs.  Linux VMs can have long names.

Reply:
Good catch. I'll revert the change.